common/src/main/java/com/microsoft/identity/common/internal/broker/BrokerValidator.java

```diff
@@ -229,6 +231,26 @@ private X509Certificate getSelfSignedCert(final List<X509Certificate> certs)
         return selfSignedCert;
     }
 
+    /**
+     * Returns the package that is currently active relative to the Work Account custom account type
+     * Note: either the company portal or the authenticator
+     *
+     * @param context : current active broker package name, null if no broker is available
```

Comment:
Point of curiosity: what's the relationship between this context and mContext?

Reply:
Good catch, didn't realize we already have a member variable for `Context`, removed `context` as an arg.